astroid/bases.py

```diff
@@ -87,8 +91,13 @@ def _is_property(meth, context=None):
     if not meth.decorators:
         return False
     for decorator in meth.decorators.nodes or ():
-        inferred = helpers.safe_infer(decorator, context=context)
-        if inferred is None or inferred is util.Uninferable:
+        # Circular import
+        from astroid.helpers import (  # pylint: disable=import-outside-toplevel
+            safe_infer,
+        )
```

Comment:
We should avoid putting an import inside a `for` loop.

Reply:
I did that change too fast, you're damn right.